packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -26,6 +27,16 @@ class ConfigCommand extends FlutterCommand {
       negatable: false,
       hide: !verboseHelp,
       help: 'Print config values as json.');
+    for (Feature feature in allFeatures) {
```

Comment:
You'd want to complete the feedback loop and make these settings actionable by setting session values on google analytics

Reply:
I've wired them up in a comma separated list.